src/app/job-registration-form/job-registration-form.component.spec.ts

```diff
@@ -26,16 +29,111 @@ describe('JobRegistrationFormComponent', () => {
                 HttpClientTestingModule,
                 BrowserAnimationsModule
             ],
+            providers: [JobService],
             schemas: [CUSTOM_ELEMENTS_SCHEMA]
-        }).compileComponents();
-    }));
+        });
 
-    beforeEach(() => {
         fixture = TestBed.createComponent(JobRegistrationFormComponent);
         component = fixture.componentInstance;
         fixture.detectChanges();
+        control = new FormControl();
     });
-    it('should create', () => {
+
+    it('JobRegistrationFormComponent should create', () => {
         expect(component).toBeTruthy();
     });
+
+    it('createForm should create jobForm', () => {
+        component.jobForm = null;
+        component.createForm();
+        expect(component.jobForm.getRawValue().toString()).toBe(
+            {
+                date: null,
+                idea: '',
+                organisation: '',
+                region: '',
+                category: '',
+                email: '',
+                contactName: '',
+                website: '',
+                phone: '',
+                description: ''
+            }.toString()
+        );
+    });
+
+    it('noWhiteSpaceValidator function should return that empty string is invalid', () => {
```

Comment:
add ``describe`` block before all tests for "noWhiteSpaveValidator". And replace word "function" to "#" prefix. -> ``describe("#noWhiteSpaveValidator should", () => { ``